spring-cql/src/test/java/org/springframework/cassandra/test/integration/core/template/async/Book.java

```diff
@@ -9,7 +9,7 @@ public static final String uuid() {
 	}
 
 	public static Book random() {
-		return new Book(uuid(), uuid());
+		return new Book("title-" + uuid(), "isbn-" + uuid());
```

Comment:
Since this factory method is creating a single instance of Book, perhaps the UUID should be the same, as in...

```
public static Book random() {
  String uuid = uuid();
  return new Book("title - " + uuid, "isbn - " + uuid);
}
```